package.json

```diff
@@ -25,7 +25,7 @@
     "pack:sdk": "cd dist/ama-sdk && npm pack",
     "package:sdk": "npm run build:sdk && npm run pack:sdk",
     "06": "echo ------------------------------------------- E2E tests ----------------------------------------------",
-    "pree2e": "npm run compile-less",
+    "pree2e": "npm run compile-less && webdriver-manager update --gecko=false",
```

Comment:
No need to invocate the webdriver-manager update script here. Please check the angular cli options for e2e. There is an option for it.

Reply:
Indeed the update is covered by default in ng e2e. Thank you.